package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "bundle": "node fix-style-spec && browserify --standalone MigrationTool index.js > build/bundle.js",
+    "bundle": "node fix-style-spec && browserify --standalone MigrationTool index.js > bundle.js",
```

Comment:
Ah, the `bundle.js` at the root shouldn't have been committed, and it's the watchify task that's wrong.  I'll fix.